<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrogram Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="specFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="paramFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="paramButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Parameters</string>
     </property>
    </widget>
    <widget class="QFrame" name="paramEditFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>241</width>
       <height>221</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="rateLabel">
         <property name="text">
          <string>Sampling Rate (Hz)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="rateLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="fftLabel">
         <property name="text">
          <string>FFT Size (samples)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fftLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="hopLabel">
         <property name="text">
          <string>Hop Size (samples)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="hopLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lengthLabel">
         <property name="text">
          <string>Window Length (samples)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lengthLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="windowingLabel">
         <property name="text">
          <string>Window Function</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="windowingComboBox">
         <item>
          <property name="text">
           <string>Hann</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hamming</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rectangular</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="scaleLabel">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="scaleComboBox">
         <item>
          <property name="text">
           <string>Mel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Logarithmic</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="dimensionLabel">
         <property name="text">
          <string>Mel Bands</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="dimensionLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="createButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Spectrogram</string>
      </property>
     </widget>
    </widget>
    <zorder>paramEditFrame</zorder>
    <zorder>paramButton</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_Parameters">
     <property name="title">
      <string>Export Parameters</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuExport_Spectrogram">
     <property name="title">
      <string>Export Spectrogram</string>
     </property>
     <addaction name="actionSavePng"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="menuExport_Spectrogram"/>
    <addaction name="menuExport_Parameters"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionImport_Audio">
   <property name="text">
    <string>Import Audio</string>
   </property>
  </action>
  <action name="actionExport_Diagram">
   <property name="text">
    <string>Export Spectrogram</string>
   </property>
  </action>
  <action name="actionExport_Parameters">
   <property name="text">
    <string>Export Parameters</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import Audio</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy to Clipboard</string>
   </property>
  </action>
  <action name="actionSaveTxt">
   <property name="text">
    <string>Save as TXT</string>
   </property>
  </action>
  <action name="actionSavePng">
   <property name="text">
    <string>Save as PNG</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
